runtime/laos/src/configs/identity.rs

```diff
@@ -37,5 +42,5 @@ impl pallet_identity::Config for Runtime {
 	type PendingUsernameExpiration = PendingUsernameExpiration;
 	type MaxSuffixLength = MaxSuffixLength;
 	type MaxUsernameLength = MaxUsernameLength;
-	type WeightInfo = (); // moonbeam_weights::pallet_identity::WeightInfo<Runtime>;
+	type WeightInfo = pallet_identity::weights::SubstrateWeight<Runtime>; // TODO is not recommended use default weights but currently there is a function within the benchmarks pallet code that cause a panic: https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/identity/src/benchmarking.rs#L608
```

Comment:
I recommend briefly explaining the reason why the benchmark is failing, as source code is changing and the link might be invalid one day